modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/SubtaskOutlet.tsx

```diff
@@ -60,12 +68,26 @@ const SubtaskOutlet = () => {
 		(response) => testraySubtaskIssuesImpl.transformDataFromList(response)
 	);
 
+	const {data: testraySubtaskToSplit} = useFetch<APIResponse<TestraySubTask>>(
+		`${testraySubTaskImpl.resource}&filter=${searchUtil.eq(
+			'r_splitFromTestraySubtask_c_subtaskId',
```

Comment:
you're using the name of relationship, you can get a short name trough transform data, but you need to implement this on TestraySubtask.ts file.